logic/transaction.js

```diff
@@ -256,11 +268,13 @@ Transaction.prototype.getBytes = function(
 };
 
 /**
- * Calls `ready` based on transaction type (see privateTypes)
+ * Calls `ready` based on transaction type (see privateTypes).
+ *
  * @see privateTypes
- * @param {transaction} transaction
- * @param {account} sender
- * @return {function|boolean} calls `ready` | false
+ * @param {transaction} transaction - Description of the param
+ * @param {account} sender - Description of the param
+ * @returns {function|boolean} calls `ready` | false
```

Comment:
This is confusing: it returns the result of calling the function, not the function itself. Unless it's a HOF?

Reply:
Maybe it makes sense to review existing contents of the docs again, this PR focuses more on introducing consistent code styles. I'm not really reviewing types of values, as long as there is a type defined, I assume it is the right type.